### ISSUE TITLE
problems sending byte[]

### ISSUE THREAD

**Issue by LloydPickering** [Olivine-Labs/Alchemy-Websockets#29]
Hi,

Firstly great library, it's far simpler to get going than some of the alternatives, so major kudos for that.

I am having a little problem sending byte[] from s->c though.

When using UserContext.Send(Byte[]) I receive an error client side (with 'Could not decode a text frame as UTF-8'. shown in Chrome's console).

If I send it through as System.Convert.ToBase64String(Byte[]) it comes through just fine.

My clientside is setup like:

``` javascript
var connection = new WebSocket("ws://hostname:port/");
connection.binaryType = "arraybuffer";
connection.onmessage = function (e) {
    if (event.data instanceof ArrayBuffer)
    {
    } else {
    }
}
```

Where am I going wrong? 

I'd really rather not have to base64 encode/decode strings when I have everything setup for dealing with ArrayBuffer and TypedArrays (WebGL).

**Comment by ArthurGodoy** [comment]
Same for me. Anyone knows how send binary data??

**Comment by ajacksified** [comment]
Will be taking a look soon - was on vacation last week (we need more developer redundancy, apparently)

**Comment by LloydPickering** [comment]
Just wondering if anyone is still taking a look at this?

**Comment by ajacksified** [comment]
@LloydPickering We've been totally slammed lately, but would love a pull request. We'll be getting to it soon, but I can't commit to an eta until we've got some other things finished first.

**Comment by DorianGray** [comment]
Our most recent refactor broke this. sending a byte array right now breaks because it tries to send it as string data, not binary. I'll add separate methods for sending binary data eventually...

Or you can! I'd encourage you to make a pull request implementing this feature.

**Comment by BradDodi** [comment]
Yeah, I'm having the same problem sending byte arrays. Does anyone know of a fix/workaround? 

Like LloydPickering, I don't really want to convert the byte array to a base64 string before sending each time. Thanks! :)

**Comment by ajacksified** [comment]
fixed - pull request #36
